models/service/intermediate/int_hubspot__ticket_calendar_spine.sql

```diff
@@ -0,0 +1,70 @@
+-- depends_on: {{ source('hubspot', 'ticket') }}
```

Comment:
Is this still necessary if we are using a source? I thought the depends on was only needed for references. It doesn't hurt to keep it here, but was more curious than anything about the need for this comment config.

Reply:
good question -- i had it in there because [zendesk](https://github.com/fivetran/dbt_zendesk/blob/main/models/utils/int_zendesk__calendar_spine.sql) selects from the source and still has the depends_on comment, but [jira](https://github.com/fivetran/dbt_jira/blob/main/models/intermediate/field_history/int_jira__issue_calendar_spine.sql#L73) does not and appears to work fine

Reply:
okie verified that we're good to remove this comment

Reply:
i think it's leftover from when we tried to use the staging model instead of the source